cli/tests/data/helloworld-v3-stub.json

```diff
@@ -0,0 +1,39 @@
+{
+  "constraints": [
+    [
+      "hostname",
+      "UNIQUE"
+    ]
+  ],
+  "healthChecks": [
+    {
+      "timeoutSeconds": 5,
+      "protocol": "HTTP",
+      "portIndex": 0,
+      "path": "/repo-empty-v3.json",
+      "maxConsecutiveFailures": 3,
+      "intervalSeconds": 30,
+      "gracePeriodSeconds": 120
+    }
+  ],
+  "container": {
+    "volumes": [],
+    "docker": {
+      "portMappings": [
+        {
+          "protocol": "tcp",
+          "hostPort": 8086,
+          "containerPort": 80
+        }
+      ],
+      "image": "mesosphere/universe-server:20170331T221320Z-cli-test-v3-helloworld-c3e49742c9",
+      "network": "BRIDGE"
+    },
+    "type": "DOCKER"
+  },
+  "requirePorts": true,
+  "mem": 128,
+  "cpus": 0.1,
+  "instances": 1,
+  "id": "/helloworld-universe"
+}
```

Comment:
This is one of the important changes. This adds a Universe repo that contains a helloworld package.